LICENSE

```diff
@@ -1,6 +1,9 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Andrew J. Gillis
+Copyright for portions of project gammazero/workerpool are held by
+Andrew J. Gillis, 2016 as part of project gammazero/workerpool. All
+other copyright for project highlight-run/workerpool are held by Highlight
```

Comment:
This is specific to highlight-run so would not be suitable for merge into this repo.